modules/create-policy-categories.adoc

```diff
@@ -5,14 +5,14 @@
 [id="create-policy-categories_{context}"]
 = Creating policy categories during policy creation
 
-You can create new policy categories from the system policies view.
+You can create new policy categories from the system policies view. This option is only available in {product-title-short} version 3.74 and earlier.
```

Comment:
Do you mean "and later"? Also in 3.74 it's only if you opt-into Postgres. So maybe 4.0+?

Reply:
So this is for the old way of making policy categories while creating policies - so it would be for 3.74 (non-postgres) and earlier, right? It looks like you can no longer create policies except by using the policy category tab so this isn't supported any longer?

Reply:
Correct, creating categories inline (in the policy create/edit workflow) is no longer supported in the postgres flavors.

Reply:
Removed section since this is 4.1 doc where that is no longer supported.